template/project.json

```diff
@@ -1,20 +1,29 @@
 {
-    "version": "1.0.0-*",
-    "compilationOptions": {
-        "emitEntryPoint": true
-    },
-
-    "compilerName": "fsc",
-    "compileFiles": [
-        "Program.fs"
-    ],
+  "version": "1.0.0-*",
+  "compilationOptions": {
+    "emitEntryPoint": true
+  },
+  "compilerName": "fsc",
+  "compileFiles": [
+    "Program.fs"
+  ],
+  "dependencies": {
+    "Microsoft.FSharp.Core.netcore": "1.0.0-alpha-160329",
+    "Microsoft.NETCore.App": {
+      "version": "1.0.0-rc2-23931"
+    }
+  },
 
-    "dependencies": {
-        "Microsoft.FSharp.Core.netcore": "1.0.0-alpha-160318",
-        "NETStandard.Library": "1.5.0-rc2-23911"
-    },
+  "tools": {
+       "dotnet-compile-fsc": "1.0.0-*"
```

Comment:
This is pulling down the latest from `dotnet-compile-fsc` from https://dotnet.myget.org/gallery/dotnet-cli which appears to be `v1.0.0-beta-002194` posted 3 hours ago. It looks like to have NuGet dependencies that are not resolving. May need to add the feed.

estoring Tool 'dotnet-compile-fsc' for '/root/template/obj/f2b68f1d-15c1-4c9c-8791-7b225bb95320/bin/project.json' in '/root/template/obj/f2b68f1d-15c1-4c9c-8791-7b225bb95320/bin'
log  : Restoring packages for /root/template/obj/f2b68f1d-15c1-4c9c-8791-7b225bb95320/bin/project.json...
error: Unable to resolve NuGet.ProjectModel (>= 3.5.0-beta-1105) for .NETStandardApp,Version=v1.5.
error: Unable to resolve NuGet.Versioning (>= 3.5.0-beta-1105) for .NETStandardApp,Version=v1.5.
error: Unable to resolve Microsoft.Extensions.PlatformAbstractions (>= 1.0.0-rc2-20100) for .NETStandardApp,Version=v1.5.
error: Unable to resolve NuGet.Packaging (>= 3.5.0-beta-1105) for .NETStandardApp,Version=v1.5.
error: Unable to resolve NuGet.Frameworks (>= 3.5.0-beta-1105) for .NETStandardApp,Version=v1.5.
error: Unable to resolve Microsoft.Extensions.FileSystemGlobbing (>= 1.0.0-rc2-15996) for .NETStandardApp,Version=v1.5.
log  : /root/template/obj/f2b68f1d-15c1-4c9c-8791-7b225bb95320/bin/project.json
log  : Restore failed in 133431ms.